app/src/pages/Play/YourTurnContent.tsx

```diff
@@ -333,6 +350,12 @@ function YourTurnContent(props: {
             </Grid>
           )}
         </Grid>
+
+        {props.activeTurnPlayState === ActiveTurnPlayState.Playing && (
+          <div style={{ textAlign: "center", color: grey[500] }}>
+            [ Hint: Press the spacebar for "Correct", and S for "Skip" ]
```

Comment:
Do we also want to let the user know they can press "c" for correct? Or is that to be removed from  `const SHORTCUTS_COMPLETE = [" ", "c"]` to be consistent?

Reply:
Totally your call. I implemented it as a list since I figured we may want flexibility about offering multiple shortcuts, but then didn't want to overwhelm the UI. Sort of a design choice whether the shortcuts should come from simplicity (-> a spacebar) or some semantic meaning (-> "c" for correct).

Reply:
I'm cool w/proceeding as is!

@cc: for awareness of @namoscato, who may have a stronger opinion, he mentioned it in his feedback (don't think notification will go through, so will ping)